aiohttp/connector.py

```diff
@@ -781,18 +789,23 @@ def _get_ssl_context(self, req):
         if req.is_ssl():
             if ssl is None:  # pragma: no cover
                 raise RuntimeError('SSL is not supported.')
-            sslcontext = req.ssl
-            if isinstance(sslcontext, ssl.SSLContext):
-                return sslcontext
-            if sslcontext is not None:
-                # not verified or fingerprinted
-                return self._make_ssl_context(False)
-            sslcontext = self._ssl
-            if isinstance(sslcontext, ssl.SSLContext):
-                return sslcontext
-            if sslcontext is not None:
+
+            def check_ssl(ssl_ctx):
```

Comment:
The function has no closures, please rename it into `_check_ssl` and move to the module's top-level namespace